android/src/main/java/com/swmansion/rnscreens/ScreenStackHeaderConfig.java

```diff
@@ -363,7 +367,7 @@ public void setTintColor(int color) {
 
   public void setTopInsetEnabled(boolean topInsetEnabled) { mIsTopInsetEnabled = topInsetEnabled; }
 
-  public void setBackgroundColor(int color) {
+  public void setBackgroundColor(Integer color) {
```

Comment:
Also, could you change it back to int? It is the way we pass all the color props and I think it is the preferred way of doing so.

Reply:
It was int, but int does not accept null. We need null to actually not put a color.

Reply:
For the ui to look as in the example, on iOS (and now android), you need to write


`headerTranslucent: true,
      headerStyle: {backgroundColor: 'transparent'},`

Reply:
If you make it an Integer, the value of 0 is passed for `backgroundColor: 'transparent',` and -1 if nothing is passed. There is no `null` option. Am I right?
If you make it an int, the same values are passed. 
So maybe it is better to leave the int and just check `if (mBackgroundColor != -1)` instead of 0?

Reply:
let me quickly do a test and see how it behaves

Reply:
@WoLewicki - you where right. I think I put null because it clearly says what you are doing (i.e: setting the color ONLY if the prop was set), but your solution works and i've reverted it like that to be consistent.

Here are some screenshots:

![Screenshot_20200915_150323](https://user-images.githubusercontent.com/2101405/93208298-0cfebc80-f765-11ea-8e2d-2d52e3ce102f.jpg)
![Screenshot_20200915_150357](https://user-images.githubusercontent.com/2101405/93208314-11c37080-f765-11ea-99f0-217a47f7a99b.jpg)
![Screenshot_20200915_150423](https://user-images.githubusercontent.com/2101405/93208322-1556f780-f765-11ea-9c81-fcb5d9dbb868.jpg)
![Screenshot_20200915_150446](https://user-images.githubusercontent.com/2101405/93208340-17b95180-f765-11ea-881e-9b76ad68f30a.jpg)